net/shadowsocks/src/opnsense/mvc/app/models/OPNsense/Shadowsocks/General.xml

```diff
@@ -31,5 +30,21 @@
             <default>password</default>
             <Required>N</Required>
         </password>
+        <cipher type="OptionField">
+            <default>aes-256-cfb</default>
+            <Required>Y</Required>
+            <OptionValues>
+                <aes-256-cfb>aes-256-cfb</aes-256-cfb>
```

Comment:
These option labels look better in uppercase.

Reply:
Done